cedar-lean/Cedar/Thm/Validation/Typechecker/Types.lean

```diff
@@ -84,6 +84,11 @@ def InstanceOfEntityTags (data : EntityData) (entry : EntitySchemaEntry) : Prop
   | .some tty => ∀ v ∈ data.tags.values, InstanceOfType v tty
   | .none     => data.tags = Map.empty
 
+def IsValidEntityUID (entry: EntitySchemaEntry) (eid: String) : Prop :=
```

Comment:
nit: this checks the validity of an EID, not a UID, so maybe this should be called `IsValidEntityEID`?  Or, it could take an `EntityUid` instead of `eid: String`